.github/mergify.yml

```diff
@@ -0,0 +1,51 @@
+pull_request_rules:
+- name: auto-merge
+  description: >
+    Automatic merge of PRs to main
+  conditions:
+    - "#approved-reviews-by>=2"
```

Comment:
since we are effectively only 3 active developers right now, this might become a bottleneck if one is inactive. I would be fine with a single approver, or, if this is possible, with all assigned reviewers approving (so that people might want to ask also for 2 reviewers because they are unsure). 

Or if we still allow manual merge, then having a default of >=2 is fine, too.

Reply:
We can still merge manually, but let's relax to 1 for now and put it back to 2 once we have more active maitainers :)